grafana/grafana-iot-twinmaker-app
# Expose selected tag propertyName in SceneViewer

* Adds the Selected Property for an entity as well to be updated or set as the initial values.
* Enable using template variables in the selected property field of relevant queries (GetPropertyValueHistory and GetPropertyValue)

– Testing –
* Create a template variable for the selected property
* Attach it to the Scene Viewer display options
* Use the template variable in the "Selected Properties" field of the GetPropertyValueHistory queries
* Click different tags and see successful queries in other panels using the template variables
* Use a template variable in all queries with GetPropertyValueHistory and GetPropertyValue
